src/Uuid.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Xabbuh\XApi\Model;
+
+final class Uuid
+{
+    private function __construct()
+    {
+    }
+
+    /**
+     * @param string $uuid
+     * @return \Rhumsaa\Uuid\Uuid|\Ramsey\Uuid\Uuid
+     */
+    public static function fromString($uuid)
+    {
+        if (class_exists('Rhumsaa\Uuid\Uuid')) {
+            return \Rhumsaa\Uuid\Uuid::fromString($uuid);
```

Comment:
I would also prefer to add `use` statements for both classes (we can alias them as `RhumsaaUuid` and `RamseyUuid`).